examples/gossipsub-chat.rs

```diff
@@ -68,12 +139,25 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let local_peer_id = PeerId::from(local_key.public());
     println!("Local peer id: {:?}", local_peer_id);
 
+    let noise_keys = libp2p::noise::Keypair::<libp2p::noise::X25519Spec>::new()
+        .into_authentic(&local_key)
+        .expect("Signing libp2p-noise static DH keypair failed.");
+
     // Set up an encrypted TCP Transport over the Mplex and Yamux protocols
-    let transport = libp2p::development_transport(local_key.clone()).await?;
+    let transport = libp2p::tcp::TokioTcpConfig::new()
```

Comment:
How about using the `tokio_development_transport` function instead?

Reply:
The former works with async-io / async-std, the latter with Tokio.

Reply:
> The former works with async-io / async-std, the latter with Tokio.

Alright, let's change to `tokio_development_transport`!